src/docs/style/wiredtiger.css

```diff
@@ -1,3 +1,27 @@
+@media (prefers-color-scheme: dark) {
```

Comment:
Dark mode looks interesting! Do you have a screenshot/link that shows the new output?

Reply:
I've attached screenshots to the ticket.